src/tape.js

```diff
@@ -232,5 +232,44 @@ test.skip = function skip(label, htestOpt) {
 test.createHarness = createHarness;
 freeze(test);
 
-export default test;
-export { test };
+// If you want to defer the test suites and run them after
+// loading all the test modules, use:
+//   test.deferSuites();
+// and then later:
+//   await test.runDeferredSuites();
+
+let suitesToRun = undefined;
```

Comment:
sigh. more global mutable state (ambient authority).

Did you see warnings from xs tools (`xsl` I think) about this?

Is it not feasible to put this state inside `Harness`?

Reply:
Oh, sure!  I can put it in Harness.

Reply:
thanks for putting it in Harness, but I still see it at module scope.
https://github.com/dckc/tape-xs/blob/b7b1a2920374946a676f96d68de109f3cdbbdea3/src/tape.js#L11

Is that still needed?

Reply:
Thanks, I removed the dead code.